examples/03_connectivity/plot_seed_based_correlation.py

```diff
@@ -0,0 +1,151 @@
+# ----------------------------------------------
+
+# We will work with the first subject of the adhd data set.
+# adhd_dataset.func is a list of filenames. We select the 1st (0-based) subject by indexing with [0]).
+from nilearn import datasets
+
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+func_filename = adhd_dataset.func[0]
+confound_filename = adhd_dataset.confounds[0]
+
+##########################################################################
+# Note that func_filename and confound_filename are strings pointing to files on your hard drive.
+print(func_filename)
+print(confound_filename)
+
+
+#############################
+# Time series extraction
+# ----------------------
+#
+# We are going to extract signals from the functional time series in two steps. First we will extract the mean signal within
+# the **seed region of interest**. Second, we will extract the **brain-wide voxel-wise time series**.
+#
+# We will be working with one seed sphere in the Posterior Cingulate Cortex, considered part of the Default Mode Network.
+dmn_coords = [(0, -52, 18)]
+
+##########################################################################
+# We use :class:`nilearn.input_data.NiftiSpheresMasker` to extract the **time series from the functional
+# imaging within the sphere**. The sphere is centered at dmn_coords and will have the radius we pass the
+# NiftiSpheresMasker function (here 8 mm).
+#
+# The extraction will also detrend, standardize, and bandpass filter the data.
+# This will create a NiftiSpheresMasker object.
+from nilearn import input_data
+
+seed_masker = input_data.NiftiSpheresMasker(
+    dmn_coords, radius=8,
+    detrend=True, standardize=True,
+    low_pass=0.1, high_pass=0.01, t_r=2.,
+    memory='nilearn_cache', memory_level=1, verbose=0)
+
+##########################################################################
+# Then we extract the mean time series within the seed region while regressing out the confounds that
+# can be found in the dataset's csv file
+seed_time_series = seed_masker.fit_transform(func_filename, confounds=[confound_filename])
+
+##########################################################################
+# Next, we can proceed similarly for the **brain-wide voxel-wise time series**, using :class:`nilearn.in_data.NiftiMasker`
+# with the same input arguments as in the seed_masker in addition to smoothing with a 6 mm kernel
+brain_masker = input_data.NiftiMasker(
+    smoothing_fwhm=6,
+    detrend=True, standardize=True,
+    low_pass=0.1, high_pass=0.01, t_r=2.,
+    memory='nilearn_cache', memory_level=1, verbose=0)
+
+##########################################################################
+# Then we extract the brain-wide voxel-wise time series while regressing out the confounds as before
+brain_time_series = brain_masker.fit_transform(func_filename, confounds=[confound_filename])
+
+
+#############################
+# Inspecting the extracted time series
+# ----------------------
```

Comment:
Can you make the line below the title as long as the text (to be valid restructured text).